osm-fudge/levenshtein.py

```diff
@@ -0,0 +1,44 @@
+'''Levenshtein distance function
+
+Find similarity two given strings using levenshtein distance
+
+More: https://en.wikipedia.org/wiki/Levenshtein_distance
+'''
+
+import sys
+
+def find_levenshtein_distance(str1, str2):
+    '''Returns levenshtein distance between two strings
+
+    Args:
+      str1: The source string
+      str2: The destination string
+
+    Returns:
+      The minimum number of single-character edits (insertions, deletions or substitutions)
+      required source string to  destination string
+    '''
+
+    m = len(str1)
+    n = len(str2)
+
+    d = [[0 for j in range(n + 1)] for i in range(m + 1)]
```

Comment:
What you could do here (if you want to): look into numpy's ndarray's and use that instead of lists of lists here :)

What you are doing is fine, though!